src/NeuroPlots.jl

```diff
@@ -1,5 +1,1278 @@
 module NeuroPlots
 
-# Write your package code here.
 
+using GLMakie
+import Statistics: mean
+import GMT: triangulate
+
+export plot_topography
+
+"""
+    plot_topography(channelList::Vector{String}, channelValues::Vector{Number}; gridSize = 1000)
+
+Plots a topographical map of the head over the desired points 
+given by `channelList` and their assigned `channelValues`, 
+with default grid size being `gridSize = 1000`.
+
+# Example
+```jldoctest
+plot_topography(PlotTopography.my_channel, rand(59))
+```
+"""
+function plot_topography(channelList::Vector{String}, channelValues::Vector{Number}; gridSize = 1000)
```

Comment:
This method should probably be made internal (`_plot_topography`) and instead `plot_topography` should take a Tables.jl table.

(I'll help do this when I get a few minutes)